docs/configure/styling-and-css.md

```diff
@@ -22,66 +32,136 @@ Storybook is pre-configured to recognize imports for CSS files. To add global CS
+  title="Using `@storybook/nextjs`?"
+>
+
+Storybook recreates your Next.js configuration, so you can use PostCSS in your stories without any extra configuration.
+</Callout>
+</IfRenderer>
+
+## CSS pre-processors
 
+### Vite
+Vite comes with Sass, Less, and Stylus support out-of-the-box. Read more about [Vite's CSS Pre-processor support](https://vitejs.dev/guide/features.html#css-pre-processors).
+
+### Webpack
+<IfRenderer renderer='react'>
+<Callout
+  variant="info"
+  icon="📣"
+  title="Using `@storybook/nextjs`?"
+>
+
+Storybook recreates your Next.js configuration, so you can use Sass in your stories without any extra configuration.
+
+</Callout>
+</IfRenderer>
+
+However, if you're using Webpack and want to use Sass or less, you'll need some extra configuration. We recommend installing [`@storybook/addon-styling-webpack`](https://storybook.js.org/addons/@storybook/addon-styling-webpack/) to help you configure these tools. Or if you'd prefer, you can customize [Storybook's webpack configuration yourself](../builders/webpack.md#override-the-default-configuration) to include the appropriate loader(s).
+
+## CSS-in-JS
 CSS-in-JS libraries are designed to use basic JavaScript, and they often work in Storybook without any extra configuration. Some libraries expect components to render in a specific rendering “context” (for example, to provide themes), which can be accomplished with `@storybook/addon-themes`'s [`withThemeFromJSXProvider` decorator](https://github.com/storybookjs/storybook/blob/next/code/addons/themes/docs/api.md#withthemefromjsxprovider).
 
 ## Adding webfonts
 
+### `.storybook/preview-head.html`
 If you need webfonts to be available, you may need to add some code to the [`.storybook/preview-head.html`](./story-rendering.md#adding-to-head) file. We recommend including any assets with your Storybook if possible, in which case you likely want to configure the [static file location](./images-and-assets.md#serving-static-files-via-storybook-configuration).
 
-<IfRenderer renderer='angular'>
+### `.storybook/preview.ts`
+If you're using something like [`fontsource`](https://fontsource.org/) for your fonts, you can import the needed css files in your [`.storybook/preview.ts`](./overview.md#configure-story-rendering) file.
 
-## Troubleshooting
+</IfRenderer>
 
-### Styles aren't being applied with Angular
+<!-- Angular only -->
+<IfRenderer renderer="angular">
 
-The latest Angular releases introduced significant changes in configuring and styling projects. If you're working with an Angular version greater than 13 and your styles aren't being applied, you may need to check your `angular.json` and adjust the `builder` configuration to import your CSS:
+  Storybook for Angular relies on the Angular CLI to build your stories. This means that you can use any CSS preprocessor that the Angular CLI supports. You can read more about this in the [Angular CLI documentation](https://angular.io/guide/workspace-config#style-script-config).
```

Comment:
While I don't think it'll have an impact on the rendered content, we don't indent paragraphs.